app/components/article-featured-video.js

```diff
@@ -25,6 +36,20 @@ export default Component.extend({
 		this.initVideoPlayer();
 	})),
 
+	actions: {
+		/**
+		 * FIXME FEATURED VIDEO A/B TEST ONLY
+		 */
+		dismissPlayer() {
+			this.element.classList.remove(scrollClassName);
```

Comment:
Outside Ember to boost performance?

Reply:
imo we shouldn't mix global / component's context when it's not necessary